lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -149,11 +149,23 @@ OperatingSystem get operatingSystem {
 /// This is intended to be used for testing and debugging only.
 OperatingSystem? debugOperatingSystemOverride;
 
-OperatingSystem _detectOperatingSystem() {
-  final String platform = html.window.navigator.platform!;
-  final String userAgent = html.window.navigator.userAgent;
+@visibleForTesting
```

Comment:
FWIW, I stopped using `@visibleForTesting`. Making things public in the engine code is fine because the entire library becomes private later in the process (`dart:_engine`).

Reply:
This file hasn't been cleaned up yet, it uses @visibleForTesting for the other tested method, I'm going to leave this here until we remove all. Also, if we ever move to import/export we don't have to touch this again :P